pkg/common/namespace_selection.go

```diff
@@ -1,48 +1,99 @@
-// Copyright (c) 2020 Red Hat, Inc.
+// Licensed Materials - Property of IBM
```

Comment:
Yikes!

Reply:
Haha, oops. I'll need to make sure it gets taken out of `cert-policy-controller`. 🙂